healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java

```diff
@@ -65,7 +65,10 @@ public static void dicomWebRetrieveStudy(String dicomStoreName, String studyId)
     // Execute the request and process the results.
     HttpResponse response = request.executeUnparsed();
 
-    String outputPath = "study.dcm";
+    // When specifying the output file, use an extension like ".multipart."
+    // Then, parse the downloaded multipart file to get each individual
+    // DICOM file.
+    String outputPath = "study.multipart";
```

Comment:
I think your comment is probably correct english, but incorrect engineering.  Should it be `".multipart".` instead? L68 or L71 is wrong.

Reply:
You're correct. Changed to ".multipart". to be clearer.